lib/restaurant.rb

```diff
@@ -0,0 +1,13 @@
+class Restaurant
+
+  attr_reader :menu
+  def initialize(menu_data)
+    @menu = menu_data || [{ name: "Egg Roll", price: 3 },
+    { name: "Shrimp Roll", price: 4 }, 
+    { name: "Boneless Spare Ribs", price: 9 }]
```

Comment:
Menu looks delicious! Good use of hashes to contain the menu items. Hashes within array to easily access during interation below is a great idea